server/core/src/fulltext.ts

```diff
@@ -143,7 +145,7 @@ export class FullTextIndex implements WithFind {
 
     const ids: Set<Ref<Doc>> = new Set<Ref<Doc>>()
     const baseClass = this.hierarchy.getBaseClass(_class)
-    let classes = this.hierarchy.getDescendants(baseClass)
+    let classes = this.hierarchy.getDescendants(baseClass).filter((it) => !this.hierarchy.isMixin(it))
```

Comment:
Why filter for mixins is needed here?

Reply:
We could not have _class in DocIndexState to be one of mixins so they should not go into elastic, so we should skip them on search.